editor/src/components/inspector/inspector-common.ts

```diff
@@ -526,12 +527,26 @@ export function detectFillHugFixedState(
     return null
   }
 
-  const flexGrow = foldEither(
+  const flexGrowLonghand = foldEither(
     () => null,
     (value) => defaultEither(null, parseCSSNumber(value, 'Unitless')),
     getSimpleAttributeAtPath(right(element.element.value.props), PP.create('style', 'flexGrow')),
   )
 
+  const flexGrow =
+    flexGrowLonghand ??
+    foldEither(
+      () => null,
+      (value) => {
+        return foldEither(
```

Comment:
Might make more sense to use `flatMapEither` here.

Reply:
I did start with that but ended up finding this to be slightly easier to read